test/index.ts

```diff
@@ -20,6 +20,8 @@ import {
   Merge,
   NonEmptyObject,
   NonNever,
+  noop,
+  noop as erase,
```

Comment:
This line is unnecessary - you can just use the previous import.

Reply:
yes, also the name `erase` is misleading, as it suggests that the function _does something_

Reply:
I guess this file isn't used as any form of user documentation, so providing a semantic example use isn't meaningful, you're right